### ISSUE TITLE
Convert both ways.

### ISSUE THREAD

**Issue by reggi** [dylang/node-xml#5]
This is not a criticism of your work, its great, but just something I don't understand. I'm so confused as to why when someone develops a `JSON` and `XML` converter that they don't create the other conversion. So all the community has a `JSON` to `XML` parser and a `XML` to `JOSN` parser that don't work together. Do you know of any interchangeable parsers? Your library on `npm` is just `xml` and it should really be the one that does both.

**Comment by LegitTalon** [comment]
Do One Thing Well™

**Comment by buschtoens** [comment]
Do The Things You're Named After™

*Do One Thing Well™* would apply if this was named `xml-encoder` or something similiar. Since this package ~~squats~~ *only has* the name `xml` one would assume it does both.

**Comment by LegitTalon** [comment]
I agree the name is misleading.

**Comment by chovy** [comment]
+1 for converting back to json. I need to compare the outputted xml to inputted json in my unit tests.

**Comment by caiosm1005** [comment]
+1

**Comment by KaidenP** [comment]
I find the name misreading too. You should at least have a notice saying that it only encodes data in the readme

**Comment by ozyman42** [comment]
Yeah this lib should not really be named `node-xml` unless it can do everything the name implies. Too late to change it now though.
